puppet/zulip/templates/uwsgi.ini.template.erb

```diff
@@ -16,6 +16,11 @@ gid=zulip
 
 stats=/home/zulip/deployments/uwsgi-stats
 
+<% if @uwsgi_rolling_restart != '' -%>
+master-fifo=/home/zulip/deployments/uwsgi-control
+lazy-apps=true
```

Comment:
Do we want a comment here explaining some details about why `lazy-apps` is coupled here?